atlasdb-container-test-utils/build.gradle

```diff
@@ -14,8 +14,6 @@ dependencies {
     implementation 'com.github.ben-manes.caffeine:caffeine'
     implementation 'com.google.guava:guava'
     implementation 'com.palantir.docker.compose:docker-compose-rule-core'
```

Comment:
Can't remove this one. Although the package name is docker-compose-rule, there are shared classes with docker-compose-extension.